ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/144_dsc_negative_no_dependant_op.robot

```diff
@@ -25,30 +25,65 @@ ${IS_NOT_PRESENT}                           1
 
 
 *** Test Cases ***
-Validate DSC and DSCI Created Without Service Mesh And Serverless Operators     #robocop:disable
+Validate DSC and DSCI Created With Errors Without Service Mesh And Serverless Operators     #robocop:disable
     [Documentation]    The purpose of this Test Case is to validate that DSC and DSCI are created
     ...                without dependant operators ((servicemesh, serverless) installed, but with errors
-    [Tags]    Operator    Sanity    Tier1    ODS-2527
+    [Tags]    Operator    Tier1    ODS-2527
+
+    Remove DSC And DSCI Resources
+    Uninstall Service Mesh Operator CLI
+    Uninstall Serverless Operator CLI
 
     Log To Console    message=Creating DSCInitialization CR via CLI
     Apply DSCInitialization CustomResource    dsci_name=${DSCI_NAME}
-    Sleep  30s     reason=wait for 1 minute until DSCI is created and reconciled
+    Sleep  45s     reason=wait for 45 seconds until DSCI is created and reconciled
 
     Log To Console    message=Creating DataScienceCluster CR via CLI
     Apply DataScienceCluster CustomResource    dsc_name=${DSC_NAME}
+    Log To Console    message=Checking DataScienceCluster conditions
     Wait Until Keyword Succeeds    6 min    0 sec
     ...    Check DataScienceCluster Conditions When Dependant Operators Are Not Installed
 
     [Teardown]    Reinstall Service Mesh And Serverless Operators And Recreate DSC And DSCI
 
+Validate DSC and DSCI Created With No Errors Without Service Mesh And Serverless Operators When Kserve Is Unmanaged    #robocop:disable
+    [Documentation]    The purpose of this Test Case is to validate that DSC and DSCI are created
+    ...                without dependant operators ((servicemesh, serverless) installed and with no errors
+    ...                because the Kserve component serving is unmanaged
+    [Tags]    Operator    Tier1
+
+    Remove DSC And DSCI Resources
+    Uninstall Service Mesh Operator CLI
+    Uninstall Serverless Operator CLI
+
+    Log To Console    message=Creating DSCInitialization CR via CLI
+    Apply DSCInitialization CustomResource    dsci_name=${DSCI_NAME}
+    Sleep  45s     reason=wait for 45 seconds until DSCI is created and reconciled
+
+    Log To Console    message=Creating DataScienceCluster CR via CLI
+    Apply DataScienceCluster CustomResource    dsc_name=${DSC_NAME}
+
+    ${result} =    Run Process    oc patch DataScienceCluster ${DSC_NAME} --type 'json' -p '[{"op" : "replace" ,"path" : "/spec/components/kserve/serving/managementState" ,"value" : "Unmanaged"}]'      #robocop:disable
+    ...    shell=true    stderr=STDOUT
+    IF    $result.rc != 0
+        FAIL    Can not patch serving.managementState field: ${result.stdout}
+    END
```

Comment:
could you use `Set Component State` keyword which is already implemented?

Reply:
@bdattoma this does not apply to this case. The Set Component State keywork will modify the spec.components.kserve.managementState field in the DSC, but in this test I am modifying 
spec.components.kserve.serving.managementState.

Reply:
okok. Could you create a new keyword? it'd make the test case style more consistent and readable